From b8862e6af1c6d022b8c182098e7deddb874ece19 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 18 Dec 2018 23:10:44 -0800
Subject: [PATCH] Do not pass null string to %s printf formatted string

Here the string is already decided to be NULL and passing
a null pointer to %s will not work

Fixes
error: '%s' directive argument is null

Upstream-Status: Submitted [https://gitlab.freedesktop.org/mobile-broadband/ModemManager/merge_requests/67]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 plugins/mtk/mm-broadband-modem-mtk.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/mtk/mm-broadband-modem-mtk.c b/plugins/mtk/mm-broadband-modem-mtk.c
index 475a63ac..541de4a5 100644
--- a/plugins/mtk/mm-broadband-modem-mtk.c
+++ b/plugins/mtk/mm-broadband-modem-mtk.c
@@ -191,7 +191,7 @@ get_supported_modes_ready (MMBaseModem *self,
 
     response = mm_base_modem_at_command_finish (MM_BASE_MODEM (self), res, &error);
     if (!response) {
-        mm_dbg ("Fail to get response %s", response);
+        mm_dbg ("Fail to get response");
         g_task_return_error (task, error);
         g_object_unref (task);
         return;
